content/en/agent/kubernetes/apm.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Kubernetes trace collection
+kind: documentation
+aliases:
+    - /agent/kubernetes/apm
+further_reading:
+- link: "agent/kubernetes/metrics"
+  tag: "documentation"
+  text: "Kubernetes Metrics"
+---
+
+## Setup
+
+To setup APM trace collection in a kubernetes cluster you need to:
+
+1. **Configure the Datadog Agent to accept traces**:
+
+    {{< tabs >}}
+{{% tab "Daemonset" %}}
+
+To enable APM trace collection in kubernetes:
+
+- Allow incoming data from port `8126` and set the `DD_APM_NON_LOCAL_TRAFFIC` and `DD_APM_ENABLED` variable to `true` in your *env* section:
+
+    ```yaml
+     # (...)
+          env:
+            # (...)
+            - name: DD_APM_ENABLED
+              value: 'true'
+            - name: DD_APM_NON_LOCAL_TRAFFIC
```

Comment:
Not needed

Reply:
Even if it's not mandatory, let's leave it here to make it more obvious.